src/app/tests/suites/TestCluster.yaml

```diff
@@ -25,7 +25,7 @@ tests:
     - label: "Send Test Not Handled Command"
       command: "testNotHandled"
       response:
-          error: 1
+          error: 0x85 # INVALID_COMMAND
```

Comment:
As a followup we probably want to expose the error code names to YAML and uses the name instead of the raw value.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/12318 to do that and will make sure it happens.

That said, that will assume that the symbolic names have the right values, of course.  ;)  So we'll need to audit that list at some point.

Reply:
https://github.com/project-chip/connectedhomeip/pull/12340 addresses this.